ContactManager2/src/components/InputTextBox.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import { ErrorMessage } from "./ErrorMessage";
+import React from "react";
+import { contactFormConfig } from "../config/contactFormConfig";
+
+export const InputTextBox = React.memo(function InputTextBox({ fieldId, value, errorMessage, updateValue }) {
```

Comment:
no need to use memo here

Reply:
This is to prevent re-rendering of input field unless there is a change in value and errorMessage.
for example if we remove this, and user enters firstName then last name and email is also re-rendered every time a character is entered.